src/BlazorWasmSdk/Tasks/Microsoft.NET.Sdk.BlazorWebAssembly.Tasks.csproj

```
@@ -1,35 +1,54 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
 
   <PropertyGroup>
     <BlazorWasmSdkRoot>$(RepoRoot)\src\BlazorWasmSdk\</BlazorWasmSdkRoot>
     <PackageId>Microsoft.NET.Sdk.BlazorWebAssembly</PackageId>
     <OutDirName>$(Configuration)\Sdks\$(PackageId)\tools</OutDirName>
   </PropertyGroup>
 
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
   <PropertyGroup>
-    <Description>SDK for building and publishing Blazor WebAssembly applications.</Description>
     <OutputType>Library</OutputType>
-    <RootNamespace>Microsoft.NET.Sdk.BlazorWebAssembly</RootNamespace>
-    <TargetFrameworks>$(SdkTargetFramework);net472</TargetFrameworks>
-  </PropertyGroup>
 
-  <PropertyGroup>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <EnableDefaultItems>false</EnableDefaultItems>
     <GenerateDependencyFile>false</GenerateDependencyFile>
     <IncludeBuildOutput>false</IncludeBuildOutput>
 
     <NoPackageAnalysis>true</NoPackageAnalysis>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <!-- MSBuild Task DLLs need to be versioned with every build -->
     <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
-  </PropertyGroup>
 
-  <PropertyGroup>
     <OutputPath>$(BaseOutputPath)</OutputPath>
     <IsPackable>true</IsPackable>
     <PackageLayoutOutputPath>$(ArtifactsBinDir)$(Configuration)\Sdks\$(PackageId)\</PackageLayoutOutputPath>
+    <NoDefaultExcludes>true</NoDefaultExcludes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>SDK for building and publishing Blazor WebAssembly applications.</Description>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.NET.Sdk.BlazorWebAssembly</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TargetFrameworks>$(SdkTargetFramework);net46</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <AdditionalContent Include="$(BlazorWasmSdkRoot)\Targets\**\*.*">
```

```suggestion
    <AdditionalContent Include="$(BlazorWasmSdkRoot)Targets\**\*.*">
```

Friends don't let friends use double slashes.